map/C:\Users\isaacgateno\Downloads\ANES_Issues_2020.csv

```diff
@@ -0,0 +1,57 @@
+"Issue","Cor_to_Party_ID","Normalized_Variance","Type","URL"
+"urban_unrest",0.641563379683091,0.773676216119635,"Security","https://raw.githubusercontent.com/Watts-Lab/deliberation-topics/main/topics/anes2020/urban_unrest.md"
```

Comment:
Would also be great to include non-normalized variance, and the number of points in the scale, just to facilitate some easy future analyses. =)